### ISSUE TITLE
Crash when `required-string-property-missing-min-length` is on and `required` is not array

### ISSUE THREAD

**Issue by duongdominhchau** [Redocly/redocly-cli#1066]
**Describe the bug**

Redoc crashes when `required-string-property-missing-min-length` is used and boolean is given to `required` (which expects an array).

Turning off that specific rule will make the crash disappear.

**Expected behavior**

An "invalid input" error should be displayed instead of crashing.

**Minimal reproducible OpenAPI snippet(if possible)**

```openapi
openapi: 3.1.0
info:
  version: 0.0.1
  title: BugDemo
components:
  schemas:
    BugDemo:
      type: object
      properties:
        id:
          type: string
          required: true # <--- This
```

**Screenshots**

![image](https://user-images.githubusercontent.com/56427890/233896679-1da3ed06-e59d-4164-87d9-9a11bc045446.png)

**Additional context**

**Comment by RomanHotsiy** [comment]
@tatomyr take a look at this issue.
This is most likely same cause as https://github.com/Redocly/redocly-cli/issues/764

**Comment by tatomyr** [comment]
The cause is not the same as in #764, but they are close enough. 

@duongdominhchau  the issue with the example above is that the `required` field must be an array (see the [docs](https://json-schema.org/understanding-json-schema/reference/object.html#required-properties)).
In your case it probably should be like this (mind the indentation too):

```yaml
openapi: 3.1.0
info:
  version: 0.0.1
  title: BugDemo
components:
  schemas:
    BugDemo:
      type: object
      properties:
        id:
          type: string
      required: [id] # <--- This
```

But the error itself is quite misleading though, I agree.

**Comment by tatomyr** [comment]
Should be fixed in `v1.0.0-beta.127`.
